config/m-nt.h

```diff
@@ -63,3 +63,24 @@
 #endif
 
 #define FLAT_FLOAT_ARRAY
+
+#if defined(_MSC_VER) && _MSC_VER < 1600
+#ifndef _UINTPTR_T_DEFINED
+#ifdef _WIN64
+typedef __int64 intptr_t;
+#else
+typedef int intptr_t;
+#endif /* WIN64 */
+#define _UINTPTR_T_DEFINED	
+#endif /* UINTPTR_T_DEFINED */
+#ifndef _INTPTR_T_DEFINED
+#ifdef _WIN64
+typedef __int64 intptr_t;	
+#else
+typedef int intptr_t;
+#endif /* WIN64 */
+#define _INTPTR_T_DEFINED	
+#endif /* _INTPTR_T_DEFINED */
+#else
+#define HAS_STDINT_H
```

Comment:
I don't understand:
- `m.h` is included before any other header (see `caml/config.h`), so `_UINTPTR_T_DEFINED` will not be defined here even if they are defined later (in `stdint.h` I guess).
- Also, by defining `HAS_STDINT_H`, you are saying it is OK to include `stdint.h`, but you are defining it precisely in the case of old compilers, where it supposedly does not exist, agree?
- Why guard both against old compilers *and* undefined `intptr_t`? It seems to me that it would be enough to guard against one of those conditions...